scripts/build.js

```diff
@@ -28,12 +28,6 @@ webpack(config).run(function(err, stats) {
 
   var openCommand = process.platform === 'win32' ? 'start' : 'open';
   console.log('Successfully generated a bundle in the build folder!');
-  console.log();
-  console.log('You can now serve it with any static server, for example:');
-  console.log('  cd build');
-  console.log('  npm install -g http-server');
-  console.log('  hs');
-  console.log('  ' + openCommand + ' http://localhost:8080');
-  console.log();
+  console.log('You can now deploy and serve it from <homepage>.');
```

Comment:
Just pushed an update for that - will work on instructions for publishing to GH Pages.